modules/testsuite/cxf-tests/pom.xml

```diff
@@ -316,11 +316,11 @@
             					<artifactId>spring-aop</artifactId>
             					<version>${test.spring.version}</version>
             				</artifactItem>
-            				<artifactItem>
+            				<!--artifactItem>
             					<groupId>org.springframework</groupId>
             					<artifactId>spring-asm</artifactId>
             					<version>${test.spring.version}</version>
-            				</artifactItem>
+            				</artifactItem-->
```

Comment:
Why is this being commented out ? and why is it commented instead of removed?

Reply:
@asoldano  I am testing if this can be removed in the latest spring version. Will remove it soon.